include/nbl/video/IGPUCommandBuffer.h

```diff
@@ -227,6 +227,25 @@ class NBL_API IGPUCommandBuffer :
             for (uint32_t i = _first + 1u; i < IGPUPipelineLayout::DESCRIPTOR_SET_COUNT; ++i)
                 _destPplnLayouts[i] = nullptr;
     }
+
+    uint32_t m_resetCount = 0;
```

Comment:
`m_resetCounter`

Reply:
I'd call it `m_resetCheckedStamp` to make it clear we're not counting resets on the commandbuffer but on the pool to which we're linked

Reply:
Resolved.